packages/root-cms/ui/pages/SettingsPage/SettingsPage.tsx

```diff
@@ -37,6 +40,72 @@ export function SettingsPage() {
             <ShareBox className="SettingsPage__section__users__sharebox" />
           </div>
         </div>
+        <div className="SettingsPage__section">
+          <div className="SettingsPage__section__left">
+            <Heading className="SettingsPage__section__left__title">
+              User Preferences
+            </Heading>
+            <Text
+              className="SettingsPage__section__body"
+              size="body-sm"
+              weight="semi-bold"
+              color="gray"
+            >
+              <p>These settings are for you only.</p>
+            </Text>
+          </div>
+          <div className="SettingsPage__section__right">
+            <div className="SettingsPage__section__userPref">
+              <div className="SettingsPage__section__userPref__description">
+                <Text
+                  className="SettingsPage__section__userPref__description__title"
+                  size="body"
+                  weight="semi-bold"
+                >
+                  Enable Lexical Editor (experimental)
+                </Text>
+                <Text
+                  className="SettingsPage__section__userPref__description__body"
+                  size="body-sm"
+                  weight="semi-bold"
+                  color="gray"
+                >
+                  <p>
+                    Replaces the{' '}
+                    <a
+                      href="https://editorjs.io/"
+                      target="_blank"
+                      rel="noopener noreferrer nofollow"
+                    >
+                      EditorJS
+                    </a>{' '}
+                    rich text editor with the{' '}
+                    <a
+                      href="https://lexical.dev/"
+                      target="_blank"
+                      rel="noopener noreferrer nofollow"
+                    >
+                      Lexical
+                    </a>{' '}
+                    version. The data saved by the Lexical version is backwards
+                    compatible but the editor may be missing a few features.
```

Comment:
suggestion: list the missing features

Reply:
i'd actually prefer to be more vague here, i think this is meant to be considered "experimental" for now, which may or may not launch if we find issues with this version of the editor.